editor/editor_system.cpp

```diff
@@ -979,18 +1007,18 @@ int EditorSystem::exportScene(const String& sceneName, const String& sceneFilePa
 
 	for (auto& filePair : toCopy)
 	{
-		tasks.push_back(std::make_shared<Task>([=, &progress]() {
+		tasks.push_back(std::make_shared<Task>([=, &progress]()
+		    {
 			progress++;
 			if (m_IsCopyFailed)
 			{
 				return;
 			}
-			m_IsCopyFailed = !OS::RelativeCopyFile(filePair.first, m_CurrExportDir + filePair.second);
-		}));
+			m_IsCopyFailed = !OS::RelativeCopyFile(filePair.first, m_CurrExportDir + filePair.second); }));
 	}
 
 	/// TODO: Fix the need for this dummy task (blocks the main thread while tasks are running)
-	tasks.push_back(std::make_shared<Task>([]() {}));
+	tasks.push_back(std::make_shared<Task>([]() { }));
```

Comment:
revert these formatting changes. The clang-format test might fail due to this. Also try using older version of extension if latest version gives error. Something like this happened to me before.